sdk/docs/iot/README.md

```diff
@@ -53,6 +53,8 @@ For more detailed, step-by-step guides to setup and run the IoT Hub certificate
 
 - Espressif ESP32: [How to Setup and Run Azure SDK for Embedded C IoT Hub Client on ESP32](https://github.com/Azure/azure-sdk-for-c/blob/main/sdk/samples/iot/aziot_esp32/readme.md)
 
+- Azure Device Update with Espressif ESP32: [How to Setup and Run Azure SDK for Embedded C ADU on Espressif ESP32](https://github.com/Azure/azure-sdk-for-c-arduino/blob/main/examples/Azure_IoT_Adu_ESP32/readme.md)
```

Comment:
This is an Arduino sample (and an external link): we should clarify that.

Reply:
That's true, that would be better. I'll add that.

Reply:
Done.